src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -275,36 +299,39 @@ All parameters are case-insensitive.</value>
     predictable mechanism for connecting to other PowerShell instances.
     Typically used with the CustomPipeName parameter on "Enter-PSHostProcess".
 
-    For example:
+    This parameter was introduced in PowerShell 6.2.
 
+    For example:
 
-    # PowerShell instance 1
-    pwsh -CustomPipeName mydebugpipe
-    # PowerShell instance 2
-    Enter-PSHostProcess -CustomPipeName mydebugpipe
+        # PowerShell instance 1
+        pwsh -CustomPipeName mydebugpipe
+        # PowerShell instance 2
+        Enter-PSHostProcess -CustomPipeName mydebugpipe
 
 -EncodedCommand | -e | -ec
 
-    Accepts a base64-encoded string version of a command. Use this parameter to
-    submit commands to PowerShell that require complex quotation marks or curly
-    braces. The string must be formatted using UTF-16 character encoding.
+    Accepts a Base64-encoded string version of a command. Use this parameter to
+    submit commands to PowerShell that require complex, nested quoting. The
+    Base64 representation must be a UTF-16 encoded string.
 
     For example:
 
-
-    $command = 'dir "c:\program files" '
-    $bytes = [System.Text.Encoding]::Unicode.GetBytes($command)
-    $encodedCommand = [Convert]::ToBase64String($bytes)
-    pwsh -encodedcommand $encodedCommand
+        $command = 'dir "c:\program files" '
+        $bytes = [System.Text.Encoding]::Unicode.GetBytes($command)
+        $encodedCommand = [Convert]::ToBase64String($bytes)
+        pwsh -encodedcommand $encodedCommand
 
 -ExecutionPolicy | -ex | -ep
 
     Sets the default execution policy for the current session and saves it in
-    the "$env:PSExecutionPolicyPreference" environment variable. This parameter
-    does not change the PowerShell execution policy that is set in the
-    registry.
+    the $env:PSExecutionPolicyPreference environment variable. This parameter
+    does not change the persistently configured execution policies.
 
--InputFormat | -inp | -if
+    This parameter only applies to Windows computers. The
+    $env:PSExecutionPolicyPreference environment variable does not exist on
+    non-Windows platforms.
+
+-InputFormat | -in | -if
```

Comment:
Per https://github.com/PowerShell/PowerShell/blob/3b83a686a1f1b8811dc0480e2b671a2577333ceb/src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs#L897 this should be `-inp`

Reply:
fixed